pkg/azuredisk/mockkubeclient/interface.go

```diff
@@ -0,0 +1,706 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mockkubeclient
+
+import (
+	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
+	discovery "k8s.io/client-go/discovery"
+	v1 "k8s.io/client-go/kubernetes/typed/admissionregistration/v1"
+	v1beta1 "k8s.io/client-go/kubernetes/typed/admissionregistration/v1beta1"
+	v1alpha1 "k8s.io/client-go/kubernetes/typed/apiserverinternal/v1alpha1"
+	v10 "k8s.io/client-go/kubernetes/typed/apps/v1"
+	v1beta10 "k8s.io/client-go/kubernetes/typed/apps/v1beta1"
+	v1beta2 "k8s.io/client-go/kubernetes/typed/apps/v1beta2"
+	v11 "k8s.io/client-go/kubernetes/typed/authentication/v1"
+	v1beta11 "k8s.io/client-go/kubernetes/typed/authentication/v1beta1"
+	v12 "k8s.io/client-go/kubernetes/typed/authorization/v1"
+	v1beta12 "k8s.io/client-go/kubernetes/typed/authorization/v1beta1"
+	v13 "k8s.io/client-go/kubernetes/typed/autoscaling/v1"
+	v2beta1 "k8s.io/client-go/kubernetes/typed/autoscaling/v2beta1"
+	v2beta2 "k8s.io/client-go/kubernetes/typed/autoscaling/v2beta2"
+	v14 "k8s.io/client-go/kubernetes/typed/batch/v1"
+	v1beta13 "k8s.io/client-go/kubernetes/typed/batch/v1beta1"
+	v2alpha1 "k8s.io/client-go/kubernetes/typed/batch/v2alpha1"
```

Comment:
hi @sunpa93 where did you copy these code from? We have issue upgrading to k8s v0.21.0 lib

Reply:
nvm, it's fixed